assets/js/components/ViewOnlyMenu/SharedServices.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * ViewOnlyMenu > SharedServices component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import Service from './Service';
+const { useSelect } = Data;
+
+export default function SharedServices() {
+	const viewableModules = useSelect( ( select ) =>
+		select( CORE_USER ).getViewableModules()
+	);
+
+	return (
+		<li className="googlesitekit-view-only-menu__list-item">
+			<h4>{ __( 'Shared services', 'google-site-kit' ) }</h4>
+			<ul>
+				{ viewableModules?.map( ( moduleSlug ) => (
```

Comment:
If there are no viewable modules this is going to output a header with no content, which seems quite odd. Is this even likely to be false-y? If so, shouldn't we return `null` for the whole component instead?

Reply:
Hmm, that's a good point. I'm not sure of the likelihood of this component being rendered while the modules are still loading, but `getViewableModules` can return `undefined` in this situation so it seems reasonable to handle it here. However, as you have suggested it would be better to return `null` for the whole component in this case. I'll make the change :+1: